inc/about/namespace.php

```diff
@@ -204,7 +209,7 @@ function render_credits_page() {
 			<p>Altis includes other libraries as part of WordPress.</p>
 			<?php
 			array_walk( $libraries['data'], '_wp_credits_build_object_link' );
-			echo '<p class="wp-credits-list">' . wp_sprintf( '%l.', $libraries['data'] ) . "</p>\n\n";
+			echo '<p class="wp-credits-list">' . sprintf( '%s.', wp_kses( implode( ', ', $libraries['data'] ), [ 'a' => [ 'href' => [] ] ] ) ) . "</p>\n\n";
```

Comment:
Could use `whitelist_html` here instead now that's in Altis? Also, no need for `sprintf( '%s'`, can move the `.` back into the string at the end.

Reply:
Oh yeah duh, the problem is our coding standards aren't aware of `whitelist_html()` as an escaping function.

Reply:
Ah! Added that in HM-Minimum I think